pkg/initializer/validator/datavalidator_test.go

```diff
@@ -293,6 +298,57 @@ var _ = Describe("Running Datavalidator", func() {
 			Expect(int(dataDirStatus)).Should(Equal(DataDirectoryStatusUnknown))
 		})
 	})
+
+	Context("with inconsistent revision numbers between etcd and latest snapshot and WALs file have some uncommitted data", func() {
```

Comment:
Please move this test to line 200, before the `with inconsistent revision numbers between etcd and latest snapshot` test